dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -733,6 +734,7 @@ bool DeltaMergeStore::flushCache(const Context & context, const RowKeyRange & ra
 
 bool DeltaMergeStore::flushCache(const DMContextPtr & dm_context, const RowKeyRange & range, bool try_until_succeed)
 {
+    fiu_do_on(FailPoints::disable_flush_cache, { return true; });
```

Comment:
after enable this failpoint, it should return `false` because there is no flush cache in actual